x-pack/plugins/security_solution/common/api/detection_engine/model/schemas.ts

```diff
@@ -107,3 +110,13 @@ export const alert_tags = t.type({
 });
 
 export type AlertTags = t.TypeOf<typeof alert_tags>;
+
```

Comment:
Wonder if it's best to move these to their own file like `alert_assignees`. Might help with generating the OpenAPI specs?

Reply:
Added PR for this https://github.com/elastic/kibana/pull/172285